app/controllers/staff/transfers_controller.rb

```diff
@@ -5,10 +5,14 @@ class Staff::TransfersController < Staff::BaseController
 
   def new
     @source_activity = source_activity
+
+    authorize @source_activity, :create_transfer?
+
     @transfer = Transfer.new
     @transfer.source = @source_activity
-
-    authorize @transfer
+    if @source_activity.project? || @source_activity.third_party_project?
```

Comment:
This probably doesn't need to happen here, because `@transfer.report` won't be used to authorize the action and won't be used in the form displayed on the page. The important thing is that this association is done in the `create` action.